rainbow-me/rainbow
# Notifications: edit wallet context menu

Fixes TEAM2-80
Figma link (if any):

## What changed (plus any additional context for devs)
- converted edit wallet action sheet to context menu
- added deeplink to wallet notification settings
- react-native-ios-context-menu improvements

## Screen recordings / screenshots



## What to test
- make sure all the edit wallet menu options work properly
- make sure the context menu label is identical to the wallet label
- make sure notification settings option only appears when notifications flag is on 


## Final checklist
